help_to_heat/frontdoor/views.py

```diff
@@ -283,11 +283,7 @@ def get_context(self, request, session_id, *args, **kwargs):
         return {"council_tax_band_options": council_tax_bands}
 
     def handle_post(self, request, session_id, page_name, data, is_change_page):
-        session_data = interface.api.session.get_session(session_id)
-        if session_data.get("country") == "Scotland":
-            return redirect("frontdoor:page", session_id=session_id, page_name="benefits")
-        else:
-            return super().handle_post(request, session_id, page_name, data, is_change_page)
+        return super().handle_post(request, session_id, page_name, data, is_change_page)
```

Comment:
I don't actually know if this is needed anymore, not sure how inheritance works in python

Reply:
Nope, it's still needed